packages/quick_actions/quick_actions_platform_interface/lib/platform_interface/quick_actions_platform.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import 'package:quick_actions_platform_interface/types/types.dart';
+
+import '../method_channel/method_channel_quick_actions.dart';
+
+/// The interface that implementations of quick_actions must implement.
+///
+/// Platform implementations should extend this class rather than implement it as `quick_actions`
+/// does not consider newly added methods to be breaking changes. Extending this class
+/// (using `extends`) ensures that the subclass will get the default implementation, while
+/// platform implementations that `implements` this interface will be broken by newly added
+/// [QuickActionsPlatform] methods.
+abstract class QuickActionsPlatform extends PlatformInterface {
+  /// Constructs a QuickActionsPlatform.
+  QuickActionsPlatform() : super(token: _token);
+
+  static final Object _token = Object();
+
+  static QuickActionsPlatform _instance = MethodChannelQuickActions();
+
+  /// The default instance of [ImagePickerPlatform] to use.
+  ///
+  /// Defaults to [MethodChannelImagePicker].
```

Comment:
Comment mentions the [ImagePickerPlatform] and [MethodChannelImagePicker]. This should be updated to reflect the QuickActionsPlatform.